testcases/open_posix_testsuite/conformance/interfaces/strcmp/3-1.c

```diff
@@ -0,0 +1,85 @@
+/*
+ * This file is licensed under the GPL license. For the full content
+ * of this license, see the COPYING file at the top level of this
+ * source tree.
+ */
+
+/*
+ * assertion:
+ *
+ *	The function will compare the string pointed to by s1 to the string pointed
+ *	to by s2. Upon completion, function will return an integer less than 0,
+ *	if the string pointed to by s1 is less than the string pointed to by s2.
+ *
+ *  method:
+ *	-Generate sample string s1 with all lower case characters.
+ *	-Generate sample string s2 with all upper case characters.
+ *	-Compare the both strings using strcmp() function
+ *	-Check the return value.
+ *	-Repeat the above steps for given number of iterations.
+*/
+
+#define _XOPEN_SOURCE 700
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include "posixtest.h"
+
+#define STRING_MAX_LEN 50000
+#define STEP_COUNT 2000
+#define TRUE 1
+#define FALSE 0
+#define TNAME "strcmp/3-1.c"
+
+char *random_string(int len, _Bool is_uppercase)
```

Comment:
Please see my general input earlier on in the code review about using `tolower()`/`toupper()`, etc.